jdbcSource/README.md

```diff
@@ -0,0 +1,234 @@
+# Overview
+
+The following documentation outlines how to incorporate a JDBC Source as part of your project. This allows a user to 
+construct applications that interact with a SQL Database, and supports almost all standard SQL Commands. These interactions 
+include the ability to run queries against the aforementioned SQL Database, periodically poll the database, and use all of the 
+returned data in the given project.
+
+In order to incorporate this Extension Source, you will need to set up your local machine with a JDBC Driver that can connect 
+to your SQL Database. Once you have done this, you will need to create the Source in the VANTIQ Modelo IDE. The documentation 
+has been split into two parts, [Setting Up Your Machine](#machine) and [Setting Up Your VANTIQ Modelo IDE](#vantiq).
+
+# Prerequisites <a name="pre" id="pre"></a>
+
+**IMPORTANT:** Read the [Testing](#testing) section before building this project.
+
+An understanding of the VANTIQ Extension Source SDK is assumed. Please read the Extension Source README.md for more 
```

Comment:
would be good to make the README.md be a link to the readme (like the link in the next line)